jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -380,6 +380,15 @@ Once you have configured the environmental variables and contexts, rerun the rea
 
 image::realitycheck-pipeline.png[Screenshot showing the realitycheck project building in the CircleCI app]
 
+==== Enabling Nomad Client for External Image Repository
+To enable the Nomad client so it can download images from an external or private repo, you can implement one of the two solutions suggested below:
+
+* Keep the image registry as `Public` within `Internal network` so the Nomad client can download the images without any authentication.
```

Comment:
We don't reference an `Internal network` anywhere else, is there another name we can use that is more explicit.  Is it the VPC, whatever network the cluster is in?  Or maybe we don't need to mention it at all, that the registry is public and unauthenticated is detail enough